articles-java-jee/src/main/java/com/ibm/cns/articles/control/JPADataAccess.java

```diff
@@ -0,0 +1,98 @@
+package com.ibm.cns.articles.control;
+
+import com.ibm.cns.articles.entity.Article;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Resource;
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.UserTransaction;
+
+@ApplicationScoped
```

Comment:
Hi Adam,
I know it was not introduced by you but is @ApplicationScoped necessary?
BTW: Thx for your refactoring :-)

Reply:
I'm not done yet, it was the very first attempt. The ms is not JakartaEE-stic yet :-)